app/Http/Controllers/Actions/Approvals/ApproveApproval.php

```
@@ -18,30 +23,36 @@
+    public function approve($modelType, $model, $request)
+    {
+        $result = $model->updateApproval(["status" => RequestApprovalStatus::APPROVED, "date_approved" => Carbon::now()]);
         $nextApproval = $model->getNextPendingApproval();
         if ($nextApproval) {
-            switch ($modelType) {
-                case ApprovalModels::PROJECT_PROPOSAL_REQUEST->name:
-                    $model->notify(new RequestProposalForApprovalNotification(auth()->user()->token, $model));
-                    break;
-                default:
-                    break;
+            $nextApprover = User::find($nextApproval['user_id']);
```

PHPMD (2.15.0)

Avoid using static access to class '\App\Models\User' in method 'approve'. (undefined)

(StaticAccess)